src/test/java/com/google/cloud/spanner/pgadapter/AbortedMockServerTest.java

```diff
@@ -1018,7 +1082,10 @@ private void assertEqual(com.google.cloud.spanner.ResultSet spannerResult, Resul
             // are weird, as are potential intercalaris values.
             assertNotNull(pgResult.getDate(col + 1));
           } else {
-            assertEquals(expected, pgResult.getDate(col + 1).toLocalDate());
```

Comment:
Instead of calling `getDate(col + 1).toLocalDate()` (which internally uses the local timezone for the calculation), we should get the result directly as a `LocalDate` by calling `getObject(col + 1, LocalDate.class)`.